tests/kernel/init/testcase.yaml

```diff
@@ -0,0 +1,6 @@
+common:
+  tags: device
+tests:
+  kernel.init:
+    tags: device
+    platform_whitelist: native_posix native_posix_64 qemu_x86
```

Comment:
why is this whitelist here, unlike other kernel tests?

Reply:
I copied it from `tests/kernel/device`.